modules/nf-core/sambamba/markdup/main.nf

```diff
@@ -0,0 +1,37 @@
+process SAMBAMBA_MARKDUP {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::sambamba=1.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity//sambamba:1.0--h98b6b92_0':
+        'quay.io/biocontainers/sambamba:1.0--h98b6b92_0' }"
+
+    input:
+        tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("*.mrkDup.bam"), emit: bam
```

Comment:
```suggestion
    tuple val(meta), path("*.bam"), emit: bam
```

It's preferred to not add custom suffixes to the filename. It's better to let the user do this via the `task.ext.prefix` variable. You can always add a check to see if the input filename is the same as the output filename (and let it throw an error if it is the same) => paragraph 6 here: https://nf-co.re/docs/contributing/modules#naming-conventions